x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -76,6 +77,8 @@ export const SessionView = ({
   const [currentJumpToCursor, setCurrentJumpToCursor] = useState(jumpToCursor);
   const [currentJumpToEntityId, setCurrentJumpToEntityId] = useState(jumpToEntityId);
 
+  const [refreshSession, setRefreshSession] = useState(false);
```

Comment:
We won't need this anymore now that we are just comparing the length of the pages already existing in `processedPages`